modules/nw-multus-ipvlan-object.adoc

```diff
@@ -28,26 +28,31 @@ plugin:
 |`string`
 |The name of the CNI plugin to configure: `ipvlan`.
 
+|`ipam`
+|`object`
+|The configuration object for the IPAM CNI plugin. The plugin manages IP address assignment for the attachment definition. This is required unless the plugin is chained.
+
 |`mode`
 |`string`
-|The operating mode for the virtual network. The value must be `l2`, `l3`, or `l3s`. The default value is `l2`.
+|Optional: The operating mode for the virtual network. The value must be `l2`, `l3`, or `l3s`. The default value is `l2`.
 
 |`master`
 |`string`
-|The Ethernet interface to associate with the network attachment. If a `master` is not specified, the interface for the default network route is used.
+|Optional: The Ethernet interface to associate with the network attachment. If a `master` is not specified, the interface for the default network route is used.
 
 |`mtu`
 |`integer`
-|Set the maximum transmission unit (MTU) to the specified value. The default value is automatically set by the kernel.
-
-|`ipam`
-|`object`
-|The configuration object for the IPAM CNI plugin. The plugin manages IP address assignment for the attachment definition.
-
-Do not specify `dhcp`. Configuring IPVLAN with DHCP is not supported because IPVLAN interfaces share the MAC address with the host interface.
+|Optional: Set the maximum transmission unit (MTU) to the specified value. The default value is automatically set by the kernel.
 
 |====
 
+[NOTE]
+====
+* `ipvlan` does not allow virtual interfaces to communicate with the `master` interface. Therefore the container will not be able to reach the host by using the `ipvlan` interface. Be sure that the container joins a network that provides connectivity to the host, such as a network supporting the Precision Time Protocol (`PTP`).
+* A single `master` interface cannot be enslaved by both `macvlan` and `ipvlan`.
```

Comment:
I'm guessing `master` `enslaved` usage here is unavoidable, given: https://redhat-documentation.github.io/supplementary-style-guide/#_master_and_slave

Reply:
I took this directly from https://www.cni.dev/plugins/current/main/ipvlan/ but to be honest I am uncomfortable with terminology .  I changed, and actually I understand it better with the update.